### PR TITLE
Detect missing maker bundle and suggest installation via StubCommands

### LINKED ISSUE #16: Stub make:factory and make:story commands and enable if maker-bundle isn't installed
They simply inform the user to install `symfony/maker-bundle`.

### PR DESCRIPTION
**Pull request by jschaedl**
Fixes #16

### CODE REVIEW

File: src/Bundle/Command/StubCommand.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+declare(strict_types=1);
+
+namespace Zenstruck\Foundry\Bundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class StubCommand extends Command
```
Comment (kbond) [inline src/Bundle/Command/StubCommand.php:12]: Can you make this abstract?

File: src/Bundle/Command/StubCommand.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
```
Comment (kbond) [inline src/Bundle/Command/StubCommand.php:3]: Can you remove this from all files. I'd like to be consistent with the rest of the project.

**Review by kbond — CHANGES_REQUESTED** [review]
Awesome, thanks for working on this @jschaedl! Just a few minor changes.

**Comment by jschaedl** [comment]
Hi @kbond Requested changes applied  :-)

**Review by kbond — APPROVED** [review]

**Comment by kbond** [comment]
Thanks @jschaedl.

**Pull request merged**